awsesproxy/Dockerfile

```
@@ -0,0 +1,46 @@
+#
+# Copyright 2018 Astronomer Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+ARG GO_VERSION=1.17
+FROM golang:${GO_VERSION}-alpine AS builder
+
+ENV AWSESPROXY_DIR /go/src/github.com/abutaha/aws-es-proxy
+
+RUN set -ex \
+    && apk add --no-cache make git file
+
+WORKDIR $AWSESPROXY_DIR
+
+RUN git clone https://github.com/astronomer/aws-es-proxy.git $AWSESPROXY_DIR && git checkout 4b359c78b9052c4cb8145cb92c7d560b99512043
```

- This sha does not match the version tag 1.3 <https://github.com/astronomer/aws-es-proxy/releases/tag/v1.3>
- There is no `cd` command after `clone`
```suggestion
RUN git clone https://github.com/astronomer/aws-es-proxy.git $AWSESPROXY_DIR && \
    cd $AWSESPROXY_DIR && \
    git checkout v1.3
```

---

we not using the exact v1.3 tag that does not have go1.17 support - i have raised a PR to upstream , still not yet merged

---

it clone inside that directory - so cd int to that dir  is not needed